### PR TITLE
[css-ui] Allow outline-style:auto to be process the same as solid

### PR DESCRIPTION
**Pull request by frivoal**
In response to https://github.com/w3c/csswg-drafts/issues/9201#issuecomment-1798087632

### CODE REVIEW

File: css/css-ui/outline-auto-width-001.html [hunk 1]
```diff
@@ -6,21 +6,34 @@
...
 }
 div + div {
     outline-width: 0;
 }
+hr~ div {
+    outline-style: solid;
+}
+hr ~ div + div {
+    outline-width: 0;
+}
```
Comment (Loirooriol) [inline css/css-ui/outline-auto-width-001.html:27]: This seems useless, there is a single `<div>` after the `<hr>`

```suggestion
```

File: css/css-ui/outline-auto-width-001.html [hunk 2]
```diff
@@ -6,21 +6,34 @@
...
+    outline-width: 1em;
     width: 100px;
     height: 100px;
-    margin: 1em;
+    margin: 2em;
 }
 div + div {
     outline-width: 0;
 }
+hr~ div {
```
Comment (Loirooriol) [inline css/css-ui/outline-auto-width-001.html:22]:
```suggestion
hr ~ div {
```

File: css/css-ui/outline-auto-width-001.html [hunk 3]
```diff
@@ -6,21 +6,34 @@
 <meta name=flags content="">
 <meta name=assert content="outline-width is ignored when outline-style is auto">
 <link rel=help href="https://drafts.csswg.org/css-ui-4/#outline-style">
-<link rel="match" href="reference/outline-auto-with-001-ref.html">
+<link rel="match" href="reference/outline-auto-width-001-ref.html">
+<link rel="match" href="reference/outline-auto-width-001-b-ref.html">
```
Comment (Loirooriol) [inline css/css-ui/outline-auto-width-001.html:10]: Might be good to add a HTML comment explaining that the 2nd ref is for allowing `outline-style: auto` to behave as `none`.

File: css/css-ui/reference/outline-auto-width-001-b-ref.html [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+div {
+    outline-style: solid;
+    outline-width: 1em;
+    width: 100px;
+    height: 100px;
+    margin: 2em;
+}
+div + div {
+    outline-style: none;
+}
```
Comment (Loirooriol) [inline css/css-ui/reference/outline-auto-width-001-b-ref.html:16]:
```suggestion
```

Already set for `div { ... }`.

**Review by Loirooriol — APPROVED** [review]
Thanks!

File: css/css-ui/reference/outline-auto-width-001-b-ref.html (hunk 4 above)
Comment (Loirooriol) [reply css/css-ui/reference/outline-auto-width-001-b-ref.html:16]: Sorry, this was fine. Multitasking XD

File: css/css-ui/outline-auto-width-001.html (hunk 3 above)
Comment (frivoal) [reply css/css-ui/outline-auto-width-001.html:10]:
```suggestion
<link rel="match" href="reference/outline-auto-width-001-ref.html">
<link rel="match" href="reference/outline-auto-width-001-b-ref.html"><!-- Allowing 'outline-style: auto' to behave the same as 'solid'-->
```

**Review by CASABECI — COMMENTED** [review]
Wkn

**Pull request merged**